wire-ios-transport/Source/Public/ZMUpdateEvent.swift

```diff
@@ -224,7 +224,7 @@ open class ZMUpdateEvent: NSObject {
     /// True if the event contains cryptobox-encrypted data
     open var isEncrypted: Bool {
         switch self.type {
-        case .conversationOtrAssetAdd, .conversationOtrMessageAdd:
+        case .conversationOtrAssetAdd, .conversationOtrMessageAdd: // .conversationMLSMessageAdd?
```

Comment:
Makes sense

Reply:
Maybe this property is also not used anymore?

Reply:
Is there an mls asset add event too?

Reply:
I see only those two:
```
case conversationMLSWelcome = 41
case conversationMLSMessageAdd = 42
```

Reply:
I removed the property.